File: 프로그래머스/unrated/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.py
# KMP문제인데 알고리즘이 기억이 안난다. 그냥 브루트포스로 감
def solution(myString, pat):
    l = len(pat)
    cnt = 0
    for i in range(len(myString)-l+1):
        if myString[i:i+l] == pat:
            cnt += 1
            
    return cnt

File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.java
class Solution {
    public int solution(String myString, String pat) {
        int cnt = 0;
        for(int i=0; i < myString.length() - pat.length() + 1; i++) {
            if (myString.startsWith(pat, i)) {
                cnt += 1;
            }
        }
        return cnt;
    }
}